environment.yml

```diff
@@ -8,7 +8,7 @@ dependencies:
 - python~=3.9
 - alembic~=1.8
 - archive-path~=0.4.2
-- asyncssh~=2.19.0
+- asyncssh~=2.21.0
 - circus~=0.19.0
```

Comment:
Opened a PR with tests for 3.14
https://github.com/circus-tent/circus/pull/1233